client/components/common/FileUpload.vue

```diff
@@ -92,8 +88,17 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+.file-upload, .upload-form {
+  display: inline-block;
+}
+
+// Using width/height restriction on hidden element
+// rather than `display: none;` because of Safari (v11.1 & v11.2) issue
+// https://forums.developer.apple.com/thread/103471
 .upload-input {
   visibility: hidden;
+  max-width: 0;
+  max-height: 0;
```

Comment:
I don't know how we ended with `visiblity: hidden` here but let's not throw away years of established practices here :point_up: 
When it comes to styling file inputs with pure css there are 2 dominant schools of thought:
1. Wrap input with label and style label instead: https://bulma.io/documentation/form/file/ (easy but I assume it is not very a11y friendly)
2. Visually hide input and style label after it:
    _(I'm giving you 2 links here but they boil down to essentially same technique.)_
    - https://tympanus.net/codrops/2015/09/15/styling-customizing-file-inputs-smart-way/
    - https://benmarshall.me/styling-file-inputs/

Go ahead and pick your poison :bowtie: